Makefile

```diff
@@ -52,8 +53,44 @@ ifdef PHOTON_SRC_REPO
 else
 	$(eval BUILD_ARG_PHOTON=)
 endif
+
+ifdef UBI_SRC_REPO
+	$(eval BUILD_ARG_UBI=--build-arg ubi_src_repo=$(UBI_SRC_REPO))
+else
+	$(eval BUILD_ARG_UBI=)
+endif
 	sudo docker build -t $(BINARY_NAME_AKO):latest --label "BUILD_TAG=$(BUILD_TAG)" --label "BUILD_TIME=$(BUILD_TIME)" $(BUILD_ARG_GOLANG) $(BUILD_ARG_PHOTON) -f Dockerfile.ako .
 
+.PHONY: ako-operator-docker
```

Comment:
can we initialise these vars outside the target, globally, since they seem to be getting used for both ako-operator-docker and build.